ntbs-service/Models/Enums/Status.cs

```diff
@@ -0,0 +1,8 @@
+namespace ntbs_service.Models.Enums 
+{
+    public enum Status {

```

Comment:
Very similar to what @lbbethke has on her branch, afraid you'll be second one merging - can you make sure we only have one copy of that enum? :)